### DISCUSSION TITLE
How to decode PancakeSwap: Smart Router V3 tx input data?

### DISCUSSION THREAD

**Discussion by andrei-volkau** [Elnaril/uniswap-universal-router-decoder#26] [General]
I would like to decode PancakeSwap: Smart Router V3 tx input data. 

I am sending you a sample TX. 
https://bscscan.com/tx/0xb2e41d356b1eb77df7e250874f710181c4e122f2d1a92cb4fce5d063636db284

Could you give me any hints how to do that, please?

**Comment by Elnaril** [comment]
Hi @andrei-volkau 
Since this is not related to the Universal Router, I would suggest to reach Uniswap support.
Closing as not relevant to the lib.
